blueocean-dashboard/src/main/js/components/Pipelines.jsx

```diff
@@ -68,18 +72,29 @@ export class Pipelines extends Component {
                                 { organization && orgLink }
                             </h1>
                         </Extensions.Renderer>
+                        
+                        {fastSearchFeatureFlag &&
+                            <div className="TextInput search-pipelines-input u-icon-left" iconLeft="search">
+                                <div className="TextInput-icon u-icon-left">
+                                    <Icon icon="search" />
+                                </div>
+                                <input className="TextInput-control" value={this.getSearchText()} placeholder="Search pipelines..." onChange={(e) => {this.onChange(e.target.value ? e.target.value : '');}} />
+                            </div>
+                        }
                     </div>
                     <Extensions.Renderer extensionPoint="jenkins.pipeline.create.action">
                         <CreatePipelineLink />
                     </Extensions.Renderer>
                 </ContentPageHeader>
                 <main>
                     <article>
-                        <Extensions.Renderer
-                            extensionPoint="jenkins.pipeline.list.top"
-                            store={ this.context.store }
-                            router={ this.context.router }
-                        />
+                        {!this.getSearchText() &&
+                            <Extensions.Renderer
+                                extensionPoint="jenkins.pipeline.list.top"
+                                store={ this.context.store }
+                                router={ this.context.router }
+                            />
+                        }
```

Comment:
Not sure we want to be excluding extension points like this. Probably ok for now but really we should be providing the seaqrch test or soemthing to this extension point so that it can use it if needed. Not sure exactly.